packages/@lwc/errors/src/compiler/error-info/template-transform.ts

```diff
@@ -580,4 +580,11 @@ export const ParserDiagnostics = {
         level: DiagnosticLevel.Error,
         url: '',
     },
+    DUPLICATE_ATTR_PROP_TRANSFORM: {
+        code: 1147,
+        message:
+            'Duplicate HTML attribute to JavaScript property mapping, "{0}" and "{1}" both map to "{2}", using the value for "{1}".',
```

Comment:
@pmdartus could you review the wording for the warning message?

Reply:
What do you think about?

```suggestion
            'Found multiple HTML attributes mapping to the same JavaScript property. "{0}" and "{1}" both map to "{2}"',
```